pinc/Quiz.inc

```diff
@@ -284,10 +300,12 @@ function user_has_passed_quiz_page($username, $quiz_page_id, $pass_requirements
 
 function get_last_attempt_date_for_quiz_page($username, $quiz_page_id)
 {
-    $res = mysqli_query(DPDatabase::get_connection(), "
-        SELECT date FROM quiz_passes
-        WHERE username='$username' AND quiz_page='$quiz_page_id'
-    ");
+    $sql = sprintf("
+        SELECT *
```

Comment:
Showing my ignorance, here, but is there a reason for changing from `SELECT date` to `SELECT *` here? (Or does it not matter?)

Reply:
It doesn't actually matter since we fetch the column by name from the row. Fetching the whole row has the slight benefit that we might pull it from the MySQL query cache since the query is identical to the other queries in this file.